doc/articles/uno-development/guidelines-for-pull-requests.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Guidelines for pull-requests
+---
+
+## Reviewing
+
+Maintainers, contributors and the community can participate in reviewing pull-requests. We require `two approvals` before the pull-request can be merged. Please apply the appropriate labels to the pull-request when reviewing. If the pull-request requires minor touch ups, consider doing them in the GitHub editor rather than asking the initiator of the pull-request to do them for you.
```

Comment:
> We require `two approvals` before the pull-request can be merged.

This is a bit... aspirational. The average number of approvals for the 10 most-recently-closed PRs is 1.1.

Reply:
It is indeed aspirational but it can be done. See comments at https://github.com/unoplatform/uno/pull/1105#discussion_r299314003